src/main/java/it/pagopa/interop/probing/caller/util/ClientUtil.java

```diff
@@ -0,0 +1,88 @@
+package it.pagopa.interop.probing.caller.util;
+
+import java.io.IOException;
+import java.net.URI;
+import java.util.Objects;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Component;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import feign.Response;
+import it.pagopa.interop.probing.caller.config.client.RestClientConfig;
+import it.pagopa.interop.probing.caller.config.client.SoapClientConfig;
+import it.pagopa.interop.probing.caller.dto.EserviceContentDto;
+import it.pagopa.interop.probing.caller.dto.TelemetryDto;
+import it.pagopa.interop.probing.caller.dtos.Problem;
+import it.pagopa.interop.probing.caller.soap.probing.ObjectFactory;
+import it.pagopa.interop.probing.caller.soap.probing.ProbingResponse;
+import it.pagopa.interop.probing.caller.util.logging.Logger;
+
+@Component
+public class ClientUtil {
+
+  @Autowired
+  RestClientConfig restClientConfig;
+
+  @Autowired
+  SoapClientConfig soapClientConfig;
+
+  @Autowired
+  private Logger logger;
+
+  public TelemetryDto callProbing(EserviceContentDto service) {
+
+    TelemetryDto telemetryResult =
+        TelemetryDto.builder().eserviceRecordId(service.eserviceRecordId()).build();
+
+    logger.logMessageCallProbing(service.technology().getValue(), service.basePath()[0]);
+
+    long before = System.currentTimeMillis();
+    telemetryResult.checkTime(String.valueOf(before));
+    try {
+
+      if (service.technology().equals(EserviceTechnology.REST)) {
+        telemetryResult = callRest(telemetryResult, service, before);
+      } else {
+        telemetryResult = callSoap(telemetryResult, service, before);
+      }
+
+    } catch (Exception e) {
+      logger.logMessageException(e);
+      telemetryResult.status(EserviceStatus.KO).koReason(e.getMessage());
+    }
+
+    logger.logMessageResponseCallProbing(telemetryResult);
+
+    return telemetryResult;
+  }
+
+  private TelemetryDto callRest(TelemetryDto telemetryResult, EserviceContentDto service,
+      long before) throws IOException {
+    Response response = restClientConfig.feignRestClient()
+        .probing(URI.create(Objects.nonNull(service.basePath()) ? service.basePath()[0] : null));
```

Comment:
an empty `basePath` (that should not happen here) would be intercepted and registered as eservice `KO`, which is not correct

Reply:
what can we do? do we delete the control? it was inserted as an additional control despite the fact that it is expected to be populated

Reply:
fine by me if you keep the check, but it should be recognized as an internal error and not an EService malfunction

Reply:
so easier to eliminate it, done.